CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ There are many ways to contribute, here are some.
 
 ## Filing an issue
 
-If there's any issue you encounter or anything that needs to be fixed, feel free to [create a GitHub issue](https://github.com/apache/beam-starter-typescript/issues).
+Issues are not enabled in this project.
```

Comment:
maybe mention doing this under https://github.com/apache/beam/issues?

Reply:
+1 - that seems like the right place for this